specter-item/src/main/java/dev/spiritstudios/specter/impl/item/SpecterItem.java

```diff
@@ -17,13 +16,17 @@ public class SpecterItem implements ModInitializer {
 	public void onInitialize() {
 		ItemMetatags.init();
 
-		ResourceManagerHelper.get(ResourceType.SERVER_DATA).registerReloadListener(new ItemGroupReloader());
+		SpecterReloadableRegistries.registerSynced(
+			SpecterItemRegistryKeys.ITEM_GROUP,
+			DataItemGroup.CODEC,
+			DataItemGroup.PACKET_CODEC
+		);
 
-		ServerLifecycleEvents.END_DATA_PACK_RELOAD.register((server, serverResourceManager, success) -> reload(server.getRegistryManager()));
-		ServerLifecycleEvents.SERVER_STARTED.register((server) -> reload(server.getRegistryManager()));
+		ServerLifecycleEvents.END_DATA_PACK_RELOAD.register((server, serverResourceManager, success) -> reload());
```

Comment:
does server starting not call end data pack reload automatically?

Reply:
nope! its very dumb

Reply:
honestly might make an event that triggers on both just to prevent having to do that

Reply:
good idea, save us some pain if we need to use it in the future and accidentally forget one or the other

Reply:
ok tried to do this and i now understand why fapi didnt, i would need to mixin to 20 different places and provide inconsistent info to the event under some circumstances. seems like way more trouble than its worth. something to come back to in another pr